src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -153,14 +156,25 @@ public void shootAtComputer(int row, int col) {
     }
 
     public void shootAtPlayer() {
-        int max = 10;
+        Coordinate coor = new Coordinate(snakeRow,snakeCol);
+        playerShot(coor);
+
+        if (snakeCol == 10) {
+            snakeCol = 1;
+            snakeRow++;
+        }
+        else {
+            snakeCol++;
+        }
+
```

Comment:
We should remember to remove commented code because this code will be a part of hard mode.